app/features/feedback/shared/strategies/radial/reducer.js

```diff
@@ -0,0 +1,30 @@
+import _ from 'lodash';
+
+function isAnnotationWithinTolerance(rule, annotation) {
+  const annotationX = annotation.x;
+  const annotationY = annotation.y;
+  const feedbackX = rule.x;
+  const feedbackY = rule.y;
+  const tolerance = rule.tolerance;
+
+  const distance = Math.sqrt(
+    ((annotationY - feedbackY) ** 2) +
```

Comment:
attempted to update staged branch and failed on `**` (also line below)

Reply:
Ugh, yes, eslint doesn't like `Math.pow()` and our babel config doesn't like the exponentiation operator 😭